scenarios/test_codegen/test/SerDe_Test.res

```diff
@@ -41,10 +41,25 @@ describe("SerDe Test", () => {
       // arrayOfTimestamps: [],
     }
 
+    //Fails if serialziation does not work
     let set = DbFunctionsEntities.batchSet(~entityMod=module(Entities.EntityWithAllTypes))
-
+    //Fails if parsing does not work
+    let read = DbFunctionsEntities.batchRead(~entityMod=module(Entities.EntityWithAllTypes))
     //set the entity
-    await DbFunctions.sql->set([entity])
+    switch await DbFunctions.sql->set([entity]) {
+    | exception exn =>
+      Js.log(exn)
+      Assert.fail("Failed to set entity in table")
```

Comment:
Added these explicit fails, I made wrong assumptions about where the schema was failing